x-pack/plugins/ml/public/application/contexts/kibana/use_create_url.ts

```diff
@@ -40,17 +48,32 @@ export const useCreateAndNavigateToMlLink = (
   page: MlUrlGeneratorState['page']
 ): (() => Promise<void>) => {
   const mlUrlGenerator = useMlUrlGenerator();
+  const [globalState] = useUrlState('_g');
+
   const {
     services: {
       application: { navigateToUrl },
     },
   } = useMlKibana();
 
-  const redirectToMlPage = async (_page: MlUrlGeneratorState['page']) => {
-    const url = await mlUrlGenerator.createUrl({ page: _page });
-    await navigateToUrl(url);
-  };
+  const redirectToMlPage = useCallback(
+    async (_page: MlUrlGeneratorState['page']) => {
+      const pageState =
+        globalState?.refreshInterval !== undefined
+          ? {
+              globalState: {
+                refreshInterval: globalState.refreshInterval,
+              },
+            }
+          : undefined;
+
+      // @ts-ignore
```

Comment:
If we cannot get rid of this `ignore` in this PR, can you please add a `// TODO` comment and describe the reason TS is failing?

Reply:
Added here https://github.com/elastic/kibana/pull/76735/commits/0d83eac644d202155da42ecda4b39145a7a4fc0b